Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -885,6 +885,40 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
     throw "Did not find TestAgentConfig.exe at : $exePath. Ensure that TestAgent is installed."
 }
 
+function Locate-TestVersion()
```

Comment:
On the build agent, we use content of script to execute on test agent and not the files themselves. Therefore, if functions are not all in same file, it won't work.